packages/api/src/cronJobs.ts

```diff
@@ -0,0 +1,118 @@
+import { EntityManager } from '@mikro-orm/core';
+import { PostgreSqlDriver } from '@mikro-orm/postgresql';
+import { Contribution } from './entities/Contribution';
+import { Project } from './entities/Project';
+import { User } from './entities/User';
+import { env } from './env';
+import logger from './logger';
+import { buildQueryBodyString } from './utils/github/buildContributionQuery';
+import { buildDateQuery } from './utils/github/buildDateQuery';
+import { buildProjectsQuery } from './utils/github/buildProjectsQuery';
+import { buildUsersQuery } from './utils/github/buildUserQuery';
+
+interface ContributionResponse {
+  data: {
+    search: {
+      pageInfo: {
+        hasNextPage: boolean;
+        endCursor: string | null;
+      };
+      nodes: PullRequestContribution[];
+    };
+  };
+}
+
+export interface PullRequestContribution {
+  id: string;
+  title: string;
+  permalink: string;
+  mergedAt: string;
+  author: {
+    login: string;
+  };
+}
+
+export const contributionPoll = async (entityManager: EntityManager<PostgreSqlDriver>) => {
+  try {
+    const timeRange = new Date();
+    const today = new Date();
+    const yesterday = new Date(today);
+    yesterday.setDate(today.getDate() - 100);
+    timeRange.setHours(timeRange.getHours() - 5);
+
+    const userList = await entityManager.find(User, {
+      $or: [
+        {
+          contributionsLastCheckedAt: {
+            $lt: timeRange,
+          },
+        },
+        {
+          contributionsLastCheckedAt: {
+            $eq: null,
+          },
+        },
+      ],
+    });
```

Comment:
I'm pretty sure you can flip these and it'll still work and it should be less lines:
```suggestion
    const userList = await entityManager.find(User, {
      contributionsLastCheckedAt: {
        $or: [
          $lt: timeRange,
          $eq: null,
        ],
      },
    });
```

Can you try that and see if it works?